client/gutenboarding/store/types.ts

```diff
@@ -35,3 +35,10 @@ export interface ApiVertical {
 	preview: '';
 	preview_styles_url: string;
 }
+
+export type TailArgs< S extends ( headArg: any, ...tailArg: any[] ) => any > = S extends (
```

Comment:
`TailParameters<T>` to match [`Parameters<T>`](https://github.com/microsoft/TypeScript/blob/87cc8c4af8fb2adcde0dfe92cb6f4703236b3291/lib/lib.es5.d.ts#L1493-L1496)?
```suggestion
export type TailParameters< T extends ( head: any, ...tail: any[] ) => any > = T extends (
```